### ISSUE TITLE
ParamSpec interprets asterisk and slash as argument of unknown type

### ISSUE THREAD

**Issue by layday** [microsoft/pyright#1154]
**Describe the bug**
When decorating a function with keyword-only arguments or positional-only arguments with `ParamSpec`, the resultant function's signature contains an extra parameter of uknown type in the position of the asterisk or slash delimiter.

**To Reproduce**
Type check the snippet below.

**Expected behavior**
That the asterisk and slash would not be interpreted as arguments.

**Screenshots or Code**
If applicable, add screenshots or the text of the code (surrounded by triple back ticks) to help explain your problem.
```python
from typing import Callable, ParamSpec, TypeVar

P = ParamSpec('P')
R = TypeVar('R')

def decorator(fn: Callable[P, R]) -> Callable[P, R]:
    return fn

def foo(*, value: str) -> None:
    ...

# Type of "bar" is partially unknown
#   Type of "bar" is "(__p0: Unknown, value: str) -> None"
# Argument of type "(*, value: str) -> None" cannot be assigned to parameter "fn" of type "Callable[P, R]" in function "decorator"
#   Type "(*, value: str) -> None" cannot be assigned to type "Callable[P, R]"
#     Named parameter "value" is missing in destination
bar = decorator(foo)
# Type of "bar" is "(__p0: Unknown, value: str) -> None"
reveal_type(bar)

def baz(value: str, /) -> None:
    ...

# Type of "qux" is partially unknown
#   Type of "qux" is "(value: str, __p1: Unknown) -> None"
qux = decorator(baz)

```

**VS Code extension or command-line**
Either, 1.1.84

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next release.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.85, which I just published. It will also be included in the next release of pylance.
